<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgTttgui</class>
 <widget class="QWidget" name="dlgTttgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe C++ Test</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>20</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>20</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTC">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTR">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMR">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMC">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnML">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBC">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBR">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNew">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMessages">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
